bin/get_fulltext.py

```diff
@@ -2,16 +2,56 @@
 
 """get_fulltext CLI command"""
 
+import select
+import click
 import libfulltext
-from libfulltext.config import Config
+import libfulltext.config
+
+# Settings for click
+CLICK_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.command(context_settings=CLICK_SETTINGS)
+@click.option("-c", "--config", default=libfulltext.config.DEFAULT_CONFIG_PATH,
+              type=click.File(), help="Path to the configuration file to use.")
+@click.argument("prefixed_id", nargs=-1, default=None)
```

Comment:
just discussed offline: since this is a list of strings there should be a plurals